### DISCUSSION TITLE
SSL Certification Patch issue

### DISCUSSION THREAD

**Discussion by Moofaa** [cat-box/aws-foundry-ssl#19] [Q&A]
I saw there was a patch and followed the instructions, set up Putty and SSH, but on entering the ./certbot_cronjob.sh command I just get "Permission denied". Any help?

**I should note I just followed the guides for setup, I know next to nothing about this stuff.

**Comment by cat-box** [comment]
sounds like you may have missed running the command `sudo su` first or you missed the `chmod a+x ./certbot_cronjob.sh` command. Try running those commands again.

**Reply by Moofaa** [reply]
Well you are right, it was the chmod, but I still have an issue with permissions.
chmod: changing permissions of ‘certbot_cronjob.sh’: Operation not permitted

**Reply by cat-box** [reply]
make sure you run `sudo su` first

**Reply by Moofaa** [reply]
Got it, it was just an issue with the PEBKAC. You know how that is. Thanks for the help!
